Gulpfile.js

```diff
@@ -171,6 +170,37 @@ gulp.task("build-check-test", gulp.series(
 gulp.task("build-lint-test", gulp.task("build-check-test"));
 
 // Helper functions:
+function spawnTask(command, done, args)
+{
+    let lCP;
+    if (args !== undefined)
+    {
+        lCP = spawn(command, args, { stdio: "inherit" });
+    } 
+    else
+    {
+        lCP = spawn(command, { stdio: "inherit" });
+    }
+    
+    lCP.on("error", (err) =>
+    {
+        console.error("Failed to start subprocess.");
+        done(err);
+    });
+
```

Comment:
Delete, the point of `close` is that it handles both in one so we can get rid of this, right?

Reply:
no, we still want to catch this, since it allows for more informative error propagation.